src/layout.rs

```diff
@@ -31,6 +32,9 @@ macro_rules! constraint {
     ( == $expr:expr ) => {
         ratatui::prelude::Constraint::Length($expr)
     };
+    ( ~ $expr:expr ) => {
+        ratatui::prelude::Constraint::Fill($expr)
```

Comment:
Using the actual path here is more clear what is happening. But I'm already known for picking on prelude I guess :ghost: 

```suggestion
        ratatui::layout::Constraint::Fill($expr)
```

Reply:
Done!